CMakeLists.txt

```diff
@@ -125,6 +125,8 @@ option(Mapper_AUTORUN_SYSTEM_TESTS "Run the system tests as part of the Mapper_T
 option(Mapper_AUTORUN_MANUAL_TESTS "Run the system tests as part of the Mapper_Test target" OFF)
 mark_as_advanced(Mapper_AUTORUN_SYSTEM_TESTS Mapper_AUTORUN_MANUAL_TESTS)
 
+option(Mapper_COVERAGE_BUILD "Builds with coverage support" ${Mapper_DEVELOPMENT_BUILD})
```

Comment:
During debugging you can occasionally step into one of the internal functions, this is extremely rare (maybe happened once or twice to me) and simply stepping-in again will put you where you want. There are also some cases where the .gcda files (the ones containing line and branch counts) get corrupted (can happen at crashes, but not always) which simply means that you get a couple hundred lines of warnings printed to stderr, quite annoying but not a huge issue.

In general I would only turn this on when actually needing access to coverage data (while trying to improve it, or after some bigger changes to make sure there is no regression in percentage). So this should probably be default off.